releases/SN30_Pro+/4.01/sn30pro_plus.metainfo.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Richard Hughes <richard@hughsie.com> -->
+<!-- Copyright 2019 Mario Limonciello <mario.limonciello@dell.com> -->
+<component type="firmware">
+  <id>com.8bitdo.sn30pro_plus.firmware</id>
+  <name>SN30 Pro+</name>
+  <summary>Firmware for the 8Bitdo SN30 Pro+ Game Controller</summary>
+  <description>
+    <p>
+      Updating the firmware on your SN30 Pro+ device improves performance and
+      adds new features.
+    </p>
+  </description>
+  <provides>
+    <!-- USB\VID_2DC8&PID_6002&REV_0001 -->
+    <firmware type="flashed">64eb3e9c-e79c-50d0-984b-21a1641f37c5</firmware>
```

Comment:
these 4 lines are copied from the previous release. im not sure where this value comes from and it undoubtedly needs updated.